pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/buffer/TransactionBufferClientTest.java

```diff
@@ -311,10 +231,26 @@ public void testTransactionBufferChannelUnActive() {
     }
 
     @Test
-    public void testTransactionBufferLookUp() throws ExecutionException, InterruptedException {
+    public void testTransactionBufferLookUp() throws Exception {
         String topic = "persistent://" + namespace + "/testTransactionBufferLookUp";
-        tbClient.abortTxnOnSubscription(topic + "_abort_sub", "test", 1L, 1L, -1L).get();
-        tbClient.commitTxnOnSubscription(topic + "_commit_sub", "test", 1L, 1L, -1L).get();
+
+        Awaitility.await().until(() -> {
```

Comment:
why do we have to use this loop ?
in this test the command is supposed to work well
which kind of error may happen ?

Reply:
waiting pending ack init.

Reply:
can we explicitly wait for that initialisation ?

I have a question about how users are supposed to deal with this problem.
I know that TransactionBufferClient is an internal API.
but when we think about the user of "Transaction#commit" (and abort)...
does the PulsarClient wait automatically for this initialisation to complete or will Transaction#commit fail ?